pkg/graph/schema.resolvers.go

```diff
@@ -1640,7 +1640,9 @@ func (r *mutationResolver) UpdateSystemIntakeContractDetails(ctx context.Context
 		// set the fields to the values we receive
 		intake.ExistingContract = null.StringFromPtr(input.Contract.HasContract)
 		intake.Contractor = null.StringFromPtr(input.Contract.Contractor)
-		intake.ContractVehicle = null.StringFromPtr(input.Contract.Vehicle)
+		intake.ContractNumber = null.StringFromPtr(input.Contract.Number)
```

Comment:
@jstrang-oddball 's [comment on the previous PR](https://github.com/CMSgov/easi-app/pull/1744#discussion_r951417640): 
> I think this needs to be removed since the ticket says we want to keep the legacy data around and this would overwrite it.

Reply:
@jstrang-oddball I found the Slack discussion I was thinking of in a DM with @ClayBenson94. The main points:
* "keeping the legacy data around" is more about not doing a database migration to automatically move data between the database columns, because contract vehicles users have entered in the past might not necessarily be valid contract numbers.
* When this mutation is used and a user's changing contract data, we want to require them to update the intake to have a contract number instead of a contract vehicle. 
* Right now, this mutation is only used in the form for initially submitting an intake; the only case where it'd actually be blanking out existing data is if a user had a draft intake they were working on and had saved it with a contract vehicle before this got deployed, then went back to edit the contract info after this was deployed. So it's probably an edge case we don't need to worry about much.